projects/kubernetes/boot.sh

```diff
@@ -1,10 +1,20 @@
-#!/bin/bash -eu
+#!/bin/sh
+
+set -e
+
 : ${KUBE_PORT_BASE:=2222}
 : ${KUBE_VCPUS:=2}
-: ${KUBE_MEM:=4096}
+: ${KUBE_MEM:=1024}
 : ${KUBE_DISK:=4G}
 : ${KUBE_NETWORKING:=default}
 : ${KUBE_RUN_ARGS:=}
+: ${KUBE_EFI:=}
+
+[ "$(uname -s)" = "Darwin" ] && KUBE_EFI=1
+
+suffix=".iso"
+[ -n "${KUBE_EFI}" ] && suffix="-efi.iso" && uefi="--uefi"
```

Comment:
It could, but not everyone has firmware, and we had some reliability issues with it in CI, so decided to not default to uefi for qemu... @deitch is looking into making the ISOs dual efi/bios which would make it all simpler though.